python/ray/monitor.py

```diff
@@ -788,7 +788,8 @@ def run(self):
         required=False,
         type=str,
         default="info",
-        help="The logging level, default is INFO.")
+        help="The logging level threshold, choices=['debug', 'info', "
```

Comment:
No no, `choices` can be an argument to `add_argument`. :-) Then argument parser enforces this.